src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -265,7 +277,32 @@
                                                         (fn [input-state]
                                                           (controlled-input/set-input-value
                                                            input-state
-                                                           max-value)))))]
+                                                           max-value)))))
+        on-refresh-swap-proposal                    (fn []
```

Comment:
```suggestion
        on-refresh-swap-proposal                    (rn/use-callback
                                                                            (fn []
```

Reply:
Thanks, fixed, also included the dependencies array